src/newcharacter.cpp

```diff
@@ -2392,19 +2417,21 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
     catacurses::window w_blood;
     const auto init_windows = [&]( ui_adaptor & ui ) {
         w = catacurses::newwin( TERMY, TERMX, point_zero );
-        w_name = catacurses::newwin( 3, 58, point( 2, 5 ) );
-        w_gender = catacurses::newwin( 2, 33, point( 2, 6 ) );
-        w_location = catacurses::newwin( 2, 60, point( 80, 5 ) );
-        w_vehicle = catacurses::newwin( 2, 60, point( 80, 6 ) );
+        w_name = catacurses::newwin( 2, 42, point( 2, 5 ) );
+        w_gender = catacurses::newwin( 1, 42, point( 2, 7 ) );
+        w_location = catacurses::newwin( 1, TERMX - 92, point( 90, 5 ) );
```

Comment:
Minimum supported terminal width is 80, thus subtracting 92 when terminal size is less than that could lead to negative number window width and crash (see #45312).